modules/docs/arrow-docs/docs/docs/patterns/glossary/README.md

```diff
@@ -157,7 +157,7 @@ binding {
 ```
 
 ```kotlin:ank
-import arrow.core.extensions.option.applicative.map
+import arrow.core.extensions.option.apply.map
```

Comment:
have we found collision issues with the stdlib `apply` function when requesting the `apply()` instance explicitly from a companion? Seems like this could be potentially confusing to users.

Reply:
I hope not. I know the name can be confusing for some users but as most of the people will consume it as is in this example I guess this won't be an issue.

Reply:
@raulraja there is no issue with `apply`. It's called `invoke` in Kotlin. Scala mix-up :D

Reply:
@nomisRev I think @raulraja is talking about this one:

```kotlin
package kotlin

/**
 * Calls the specified function [block] with `this` value as its receiver and returns `this` value.
 */
@kotlin.internal.InlineOnly
public inline fun <T> T.apply(block: T.() -> Unit): T {
    contract {
        callsInPlace(block, InvocationKind.EXACTLY_ONCE)
    }
    block()
    return this
}
```